### PR TITLE
Add an indication that battery's missing

### PR DESCRIPTION
**Pull request by DasMatus**
This PR adds an indication of battery missing, since current version is just displaying 0% on battery, even though I'm running the greeter on a desktop PC. This PR is thoroughly tested and causes no issues when running.

### CODE REVIEW

**Comment by git-f0x** [comment]
I think it would be better to just not show the battery icon and its text when the system doesn't have a battery, similar to how the battery applet behaves.

**Comment by DasMatus** [comment]
It's already there, both on the locker and on the greeter side.

**Comment by DasMatus** [comment]
Fixed the conflict.

File: src/greeter.rs [hunk 1]
```diff
@@ -304,6 +304,84 @@ pub fn main() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+async fn request_message(socket: Arc<Mutex<UnixStream>>, request: Request) -> Message {
```
Comment (jackpot51) [inline src/greeter.rs:307]: Was adding this code intentional?

Comment (DasMatus) [reply src/greeter.rs:307]: No. I was just updating the fork. I added only the if statement that shows blank string.

File: src/greeter.rs
Comment (DasMatus) [inline src/greeter.rs:1]: Only the if statement on the battery text is mine.

**Review by DasMatus — COMMENTED** [review]
Just to be clear - GitHub showed me the merge conflict warning after the big alpha pre-release PR.

**Review by DasMatus — COMMENTED** [review]
Reviewed the changes. Probably some code made it through the fork update.

**Pull request closed without merging**